baker/algolia/indexExplorersToAlgolia.ts

```diff
@@ -99,17 +141,25 @@ const getExplorerRecords = async (): Promise<ExplorerRecord[]> => {
         )
         .then((results: ExplorerEntry[]) =>
             results.flatMap(({ slug, title, subtitle, blocks }) => {
-                const textFromExplorer = extractTextFromExplorer(blocks)
+                const textFromExplorer = extractTextFromExplorer(
+                    blocks,
+                    graphersUsedInExplorers
+                )
                 const uniqueTextTokens = new Set([
                     ...textFromExplorer.split(" "),
                 ])
                 const textChunks = chunkParagraphs(
                     [...uniqueTextTokens].join(" "),
                     1000
                 )
+
+                // In case we don't have any text for this explorer, we still want to index it
+                const textChunksForIteration = textChunks.length
+                    ? textChunks
+                    : [""]
                 const explorerRecords = []
                 let i = 0
-                for (const chunk of textChunks) {
+                for (const chunk of textChunksForIteration) {
```

Comment:
(not introduced in this PR)
nit: seems like a forEach would be more fitting here?

Reply:
Yeah, or `map` for that matter. Thanks! :)